en/components/layout.md

```diff
@@ -19,10 +19,90 @@ _keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI
 
 ###Usage
 Use the [**igxLayout**]({environment:angularApiUrl}/classes/igxlayoutdirective.html) directive on a container element to specify the layout
-direction for its children: horizontally with [`igxLayoutDir`]({environment:angularApiUrl}/classes/igxlayoutdirective.html#dir)`="row"` or vertically with
-[`igxLayoutDir`]({environment:angularApiUrl}/classes/igxlayoutdirective.html#dir)`="column"`.
+direction for its children:
 
-**Note**: the [`igxLayout`]({environment:angularApiUrl}/classes/igxlayoutdirective.html) directive affects the flow directions for that
+<div class="divider--half"></div>
+
+#####Horizontally with [`igxLayoutDir`]({environment:angularApiUrl}/classes/igxlayoutdirective.html#dir)`="row"`
```

Comment:
_Horizontally_ what? _Horizontally_ and _vertically_ are adverbs that should be used alongside verbs to signify their meaning.  Example: _Align Horizontally_

Also, you don't have to use square brackets. They are reserved only for links.

```suggestion
##### Align Horizontally with `igxLayoutDir`({environment:angularApiUrl}/classes/igxlayoutdirective.html#dir)`="row"`
```